src/app/application.cpp

```diff
@@ -368,7 +368,7 @@ void Application::processMessage(const QString &message)
         m_paramsQueue.append(params);
 }
 
-void Application::runExternalProgram(const BitTorrent::Torrent *torrent) const
+void Application::runExternalProgram(const QString &cmdLine, const BitTorrent::Torrent *torrent) const
```

Comment:
```suggestion
void Application::runExternalProgram(const QString &extProgramLine, const BitTorrent::Torrent *torrent) const
```
Please rename it to avoid confusing with Windows cmd.exe

Reply:
> Please rename it to avoid confusing with Windows cmd.exe

Do you really associate the command line with `cmd.exe` only?

Reply:
Replaced with `programTemplate`.